### DISCUSSION TITLE
Is there a way to setup `SLUG_PRESERVE_UNICODE` like in the svelte.dev website?

### DISCUSSION THREAD

**Discussion by nazarepiedady** [sveltejs/kit#10338] [Q&A]
### Describe the problem

I am on work of translating the svelte kit documentation into Portuguese,
but I am without a way to preserve the slugs in the pages generated.

What can I do to set up `SLUG_PRESERVE_UNICODE` like in the `svelte.dev` website?

### Describe the proposed solution

```js
export const SLUG_PRESERVE_UNICODE = false;
export const SLUG_SEPARATOR = '_';
````

### Alternatives considered

https://github.com/sveltejs/svelte/blob/master/sites/svelte.dev/config.js

### Importance

would make my life easier

**Comment by ghostdevv** [comment]
From what I can tell it originally came from https://github.com/sveltejs/action-deploy-docs - not sure if it's still even being used. Maybe @PuruVJ will know

**Reply by PuruVJ** [reply]
Yeah, not being used. Didn't even know this file was there 😁

**Comment by PuruVJ** [comment]
Oof I didn't even know this was there! What does it do exactly and what behavior do you expect from it?

**Reply by nazarepiedady** [reply]
@PuruVJ I am translating the svelte kit documentation, so I do not want to break the application when I edit the headings because the svelte kit documentation application generates the heading links automatically by its own algorithm.

**Reply by PuruVJ** [reply]
IG write the documentation, and if it breaks, lemme know! Will help to fix

**Reply by nazarepiedady** [reply]
@PuruVJ thanks for your kindness, I will translate so when something goes wrong I will let you know.
